earth_enterprise/rpms/build.gradle

```diff
@@ -62,15 +90,18 @@ if (new File('build.properties').exists()) {
     }
 }
 
-def openGeeVersionFile = new File("../src/NATIVE-DBG-x86_64/gee_long_version.txt")
+// Create a file handle for gee_long_version_txt
+def openGeeVersionFile = new File( gee_long_version_txt )
 
-if (!openGeeVersionFile.exists()) {
+// Call scons to create the file 'gee_long_version.txt'
+// The actual parameter used by scons can remain 'release=1'
```

Comment:
This second comment is wrong.  More details are below.

Reply:
@tst-dmendyke: This minor comment also stands; `release=1` isn't being passed to `scons` any more.